.sdkmanrc

```diff
@@ -1,3 +1,3 @@
 # Enable auto-env through the sdkman_auto_env config
 # Add key=value pairs of SDKs to use below
-java=11.0.13-librca
+java=11.0.17-librca
```

Comment:
This was bumped to try to fix sdkman issues. Didn't help but I thought it was worth updating since the older version doesn't seem to appear on sdkman versions list